sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/samples/TimeSeriesInsightsClientSample/InstancesSamples.cs

```diff
@@ -22,7 +22,8 @@ public async Task RunSamplesAsync(TimeSeriesInsightsClient client)
             PrintHeader("TIME SERIES INSIGHTS INSTANCES SAMPLE");
 
             // Figure out what keys make up the Time Series Id
-            TimeSeriesModelSettings modelSettings = await client.ModelSettings.GetAsync();
+            TimeSeriesInsightsModelSettings modelSettingsClient = client.GetModelSettingsClient();
```

Comment:
I wonder if we need the Client at the end of the name. Reason being I remember that something shouldn't be called client unless a user has to instantiate it. I believe this was a comment on the first revision. Let's keep it the way you have and get feedback from them.

Reply:
Sure, sounds good.